src/Setup_geometry.jl

```diff
@@ -1168,37 +1184,37 @@ Parameters
 end
 
 """ 
-    Compute_ThermalStructure(Temp, X, Y, Z, s::McKenzie_subducting_slab)
+    Compute_ThermalStructure(Temp, X, Y, Z, Phase, s::McKenzie_subducting_slab)
 
 Compute the temperature field of a `McKenzie_subducting_slab`. Uses the analytical solution
 of McKenzie (1969) ["Speculations on the consequences and causes of plate motions"]. The functions assumes
 that the bottom of the slab is the coordinate Z=0. Internally the function shifts the coordinate. 
 Parameters
 =============================
 Temp Temperature array
-X    X Array 
-Y    Y Array 
-Z    Z Array 
-Phase Phase array 
-s    McKenzie_subducting_slab
+- `X`    X Array 
+- `Y`    Y Array 
+- `Z`    Z Array 
+- `Phase` Phase array 
+- `s`    McKenzie_subducting_slab
 
 """
 function Compute_ThermalStructure(Temp, X, Y, Z,Phase, s::McKenzie_subducting_slab)
     @unpack Tsurface, Tmantle, Adiabat, v_cm_yr, κ, it = s
 
     # Thickness of the layer: 
-    D0          =   (maximum(Z)-minimum(Z));
+    Thickness          =   (maximum(Z)-minimum(Z));
     Zshift      =   Z .- Z[end]       # McKenzie model is defined with Z = 0 at the bottom of the slab
 
     # Convert subduction velocity from cm/yr -> m/s; 
     convert_velocity = 1/(100.0*365.25*60.0*60.0*24.0);
```

Comment:
Does this assume that the veolocity is always needed in cm/yr?  isnt it better to default to SI?